Apps/WebClient/src/ClientApp/src/components/timeline/immunization.vue

```diff
@@ -19,6 +19,38 @@ export default class ImmunizationTimelineComponent extends Vue {
     private get entryIcon(): IconDefinition {
         return faSyringe;
     }
+
+    private displayProductList(
+        immzAgents: ImmunizationAgent[],
+        providerOrClinic: string
+    ): boolean {
+        if (immzAgents.length > 0) {
+            let countEmpty = 0;
+            immzAgents.forEach((agent) => {
+                if (
+                    agent.productName == "" &&
+                    agent.lotNumber == "" &&
+                    providerOrClinic == ""
+                ) {
+                    countEmpty++;
+                }
+            });
+            if (countEmpty == immzAgents.length) return false;
+        } else return false;
+        return true;
+    }
+    private displayProduct(
+        agent: ImmunizationAgent,
+        providerOrClinic: string
+    ): boolean {
+        if (
+            agent.productName == "" &&
```

Comment:
These are truthy checks, instead use triple equals for equality checking

Reply:
code updated per @ninosamson 's confirmation on showing "na"